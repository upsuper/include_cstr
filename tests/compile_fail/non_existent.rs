use include_cstr::include_cstr;

fn main() {
    let _ = include_cstr!("non-existent");
}
