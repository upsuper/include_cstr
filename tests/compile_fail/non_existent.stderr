error: No such file or directory (os error 2)
 --> $DIR/non_existent.rs:4:27
  |
4 |     let _ = include_cstr!("non-existent");
  |                           ^^^^^^^^^^^^^^
